### ISSUE TITLE
ICEBOX: incorporate approved video links into PSD manual

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3043]
Reference: [#2999 lucid chart](https://lucid.app/lucidchart/0cd93668-a02d-4176-b97d-4f342985a57f/edit?invitationId=inv_3462b1a9-3207-4813-87e1-993924811a3f&page=0_0#)

## Definition of Done

- [ ] Ask SMEs about unknown/unclear posts from training_workflow channel.
  - [ ] [2/26/23 2023 Agilest SAFe 5.1 for Government Teams](https://teams.microsoft.com/l/message/19:d15133fbfb4d4c3a8c81701292b1890d@thread.skype/1676555978961?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1676555978961&teamName=teampsd_vha&channelName=training_workflow&createdTime=1676555978961&allowXTenantAccess=false) - Jenn to ask Lzim
  - [ ] [2/13/23 F2F Roadmap feb_epic](https://teams.microsoft.com/l/message/19:d15133fbfb4d4c3a8c81701292b1890d@thread.skype/1676336760477?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1676336760477&teamName=teampsd_vha&channelName=training_workflow&createdTime=1676336760477&allowXTenantAccess=false) 
  - [ ] [2/20/23 User-Centered Team PSD Workflows - User Personas](https://teams.microsoft.com/l/message/19:d15133fbfb4d4c3a8c81701292b1890d@thread.skype/1676923209290?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1676923209290&teamName=teampsd_vha&channelName=training_workflow&createdTime=1676923209290&allowXTenantAccess=false)
  - [ ] [3/13/23 Team PSD SAFe and #health_equity Minimum Viable Product](https://teams.microsoft.com/l/message/19:d15133fbfb4d4c3a8c81701292b1890d@thread.skype/1678747922278?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1678747922278&teamName=teampsd_vha&channelName=training_workflow&createdTime=1678747922278&allowXTenantAccess=false)
All meeting recordings in training_workflow are listed by year on pink stickies (left-hand side of Lucidchart):
- [ ] Get review & approval for suggested videos in several sections
- [ ] ?Rename generic-named recordings once they are mapped to correct chapter/section, using file naming conventions

**Comment by lijenn** [comment]
@ljmoody Where are we with this card? Can you please update the ZenHub details and/or bring this the 2024_feb_roadmap? Thanks!

**Comment by ljmoody** [comment]
@lijenn This is another "stale" card. 
Certain sections of the Team PSD manual were updated and many of the other remining updates were backburnered. I need someone to review and approve/reject inclusion of videos - I figure that's Lindsey? The list of videos in the #2999 Lucidchart is hefty.

**Comment by lzim** [comment]
Thanks @lijenn & @ljmoody 
If these aren't actively being worked on, but need a decision. We can bring these up at standing meetings to determine where they should be prioritized.

_I am unclear about what kind of approval you believe necessary for video inclusion in the manual?_

If they were listed, then they were identified as training video topics.
And, if they are on Teams, then folks will need permission to access the content.

_Is there something I'm missing?_ 

I'll put this in 2. Upcoming Epic for #2024_feb_epic so prioritizing this can be clarified.

Thanks!

**Comment by ljmoody** [comment]
@lzim - Good questions, and I can understand why you asked them. I hope this additional information helps:

> _I am unclear about what kind of approval you believe necessary for video inclusion in the manual?_

- Approval isn't needed for _all_ videos. 
- Some videos have personal content in them; I did my best to ID those in the [PSD safe workflows LucidChart](https://lucid.app/lucidchart/0cd93668-a02d-4176-b97d-4f342985a57f/edit?invitationId=inv_3462b1a9-3207-4813-87e1-993924811a3f&page=0_0#). For example, videos mapped to sticky 4.13.

**Additional info re: videos in PSD manual**
- A list of >40 videos I found in PSD channels is also within the Lucidchart linked above. I watched snippets and tried to capture the subject matter theme(s). Not all were listed or identified official training topics; most are from me rooting around and finding them.
- Other videos found, I've mapped to sections and question whether the videos are correct/intended to be located where I've put them. Sometimes videos spanned across several topics.
- There are videos mentioned on stickies in the Lucidchart I was unable to find. 
- This symbol <img width="57" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWNRQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b6b24cb761715fa5f66aeda107d5cabc34235d5/image.png" alt="image.png" /> is used within the Lucidchart for videos in/with questions.


In short, I'll need help mapping videos to sections, confirming ones I've mapped are correct/incorrect, finding 'missing' videos, and approval for videos with personal content.

**Comment by ljmoody** [comment]
Closing this card as not planned.
